config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root to: redirect('/users/sign_up')
```

Comment:
Imo no need for that. Root is set in different PR to welcome page.

Reply:
removed

Reply:
Cool, git catch!